odoo/addons/test_lint/tests/test_dunderinit.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from pathlib import Path
+
+from odoo.modules import get_modules, get_module_path
+from . import lint_case
+
+_logger = logging.getLogger(__name__)
+
+
+class TestDunderinit(lint_case.LintCase):
+
+    def test_dunderinit(self):
+        """ Test that __init__.py exists in Odoo modules, otherwise they won't get packaged"""
+
+        counter = 0
+
+        for mod in get_modules():
+            counter += 1
+            dunderinit_path = Path(get_module_path(mod)) / '__init__.py'
+            self.assertTrue(dunderinit_path.exists(), "Missing `__init__.py ` in module %s" % mod)
```

Comment:
Maybe use `is_file` instead in case someone's playing silly buggers and decides to call a folder `__init__.py`?